deltachat-ios/AppDelegate.swift

```diff
@@ -141,8 +142,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         }
         
         if let notificationOption = launchOptions?[.remoteNotification] {
+            self.pushToDebugArray("📡'")
             logger.info("Notifications: remoteNotification: \(String(describing: notificationOption))")
             increaseDebugCounter("notify-remote-launch")
+            performFetch(completionHandler: dummyCompletionHandler)
```

Comment:
not sure if that is working as expected ... we need to observe that for another day ...

Reply:
`didFinishLaunchingWithOptions:remoteNotification` may or may not be called before `didReceiveRemoteNotification`, might be that behavior has changed in iOS 14, cmp. https://stackoverflow.com/questions/59990130/push-notifications-sometimes-call-didfinishlaunchingwithoptions-instead-of-didre

what is still weird - i do not manage to start the background thread in performFetch() in all cases, seems esp. the call from `didFinishLaunchingWithOptions:remoteNotification` often fails.

Reply:
i will try over with DispatchQueue.global(qos: .default).async https://stackoverflow.com/questions/42452437/swift-3-0-async-dispatch-global-does-not-run